docs/admin/infrastructure.rst

```diff
@@ -1,4 +1,35 @@
 Infrastructure Setup
 ===========================================
 
-by Florian Maximilian Huber 
\ No newline at end of file
+.. Describe the setup of the infrastructure in terms of hardware, software and protocols so it can be configured by a system administrator at the client site. This include virtual machines, software packages etc. You can reuse the deployment diagram from the section Hardware/Software Mapping. Describe the installation and startup order for each component. You can reuse the use cases from the section Boundary Conditions. For example: If you have used docker reuse the Docker installation instructions from the cross project space.
+
+*****
+Themis App
+*****
+
+The app can be installed on iPads via `TestFlight`_. Therefore, it requires TestFlight to be installed on the iPad and the user
+to be part of at least one group, either via Apple-ID or via public TestFlight link. The only formal requirement is iPadOS 16.
+
+Apart from that, the iPad app can also be tested via XCode and its integrated Simulator.
+
+For future maintenance, the existing CI/CD-Pipeline and the TestFlight account is needed.
```

Comment:
Maybe include a note about the environment variables in XCode? I don't think they fit the topic perfectly, but there seems to be no better section in the table of contents to note that. I think you can pretty much just copy everything from here: https://confluence.ase.in.tum.de/display/IOS2223CIT/Authentication+Setup

Reply:
Thanks Paul, I will add that.